services/apps/alcs/src/alcs/planning-review/planning-review-document/planning-review-document.entity.ts

```diff
@@ -58,6 +58,25 @@ export class PlanningReviewDocument extends BaseEntity {
   @Column({ nullable: true, type: 'int' })
   evidentiaryRecordSorting?: number | null;
 
+  @Column({
+    type: 'text',
+    nullable: true,
+    comment:
+      'This column is NOT related to any functionality in ALCS. It is only used for ETL and backtracking of imported data from OATS. It links oats.planning_review to alcs.planning_review_document.',
+  })
+  oatsPlanningReviewId?: string | null;
+
+  @Column({
```

Comment:
Provide "select: false". Otherwise, this will be included in typeorm queries, and we do not need that there. This is only for ETL